inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/sort/util/SinkInfoUtils.java

```diff
@@ -233,4 +237,19 @@ public static void checkPartitionField(List<? extends SinkFieldBase> fieldList,
         }
     }
 
+    private static ElasticsearchSinkInfo createElasticsearchSinkInfo(ElasticsearchSinkResponse sinkResponse,
```

Comment:
Please add Java docs for static methods.

Reply:
fixed